code/modules/clothing/head/helmet.dm

```diff
@@ -199,6 +199,39 @@
 	armor_internaldamage = CLOTHING_ARMOR_MEDIUM
 	siemens_coefficient = 0.7
 
+/obj/item/clothing/head/helmet/caustic
```

Comment:
This should be a marine helmet subtype

Reply:
I was scared of all the overlays that came with it. But ok. Easy fix.